### PR TITLE
46238: Add Configuring RBAC Policies

### PR DESCRIPTION
**Pull request by jonquil2002**
SC: https://app.shortcut.com/replicated/story/43918/document-how-to-upload-a-support-bundle-to-vendor-portal-and-share-it-with-replicated

https://deploy-preview-321--replicated-docs.netlify.app/vendor/team-management-rbac-configuring

### CODE REVIEW

**Comment by jonquil2002** [comment]
@cremerfc  Please do a SME review on this topic, particularly the example code. Thanks!

File: docs/vendor/team-management-rbac-configuring.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Configuring Custom RBAC Policies
+
+You can configure custom role-based access control (RBAC) policies if you are on the Enterprise pricing plan. Creating custom RBAC policies lets you limit which areas of the Replicated vendor portal are accessible to team members, and control read and read/write privileges to groups based on their role. For example, you can limit access for the sales team to one application and to specific channels.
+
+:::note
+By default, every team has two policies created automatically: **Admin** and **Read Only**, regardless of which plan you are on. These default policies are not configurable.
```
Comment (paigecalvert) [inline docs/vendor/team-management-rbac-configuring.md:6]: Suggestion: Can delete "regardless of which plan you are on" as this is already assumed with "By default, every team"

File: docs/vendor/team-management-rbac-configuring.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+:::
+
+To configure custom RBAC policies:
+
+1. From the vendor portal [Team page](https://vendor.replicated.com/team), select **RBAC** from the left menu.
+
+1. Do _one_ of the following actions:
```
Comment (paigecalvert) [inline docs/vendor/team-management-rbac-configuring.md:13]: Suggestion: can delete the word "actions"

File: docs/vendor/team-management-rbac-configuring.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            "kots/app/appID/channel/channelID/list",
+            "kots/app/appID/channel/channelID/read"
+          ],
+          "denied": []
+        }
+      }
+    }
+    ```
+
+    - To get the application ID, use the [`replicated app ls`](/../reference/replicated-cli-app-ls) command or click **Settings > Show Application ID (Advanced)** in the vendor portal.
```
Comment (paigecalvert) [inline docs/vendor/team-management-rbac-configuring.md:45]: Recommend adding a stem sentence before this list that tells you what it is a list of. It took me some scrolling back up and re-reading to decide it was a list of instructions on getting necessary info from your own vendor portal team in order to use the example yaml above.

For example, "The example above uses an application ID and a channel ID to scope the permissions of the RBAC policy. To find an application ID and channel ID:"

**Review by paigecalvert — DISMISSED** [review]
Looks awesome! Just a few suggestions @jonquil2002

File: docs/vendor/team-management-rbac-configuring.md (hunk 1 above)
Comment (jonquil2002) [reply docs/vendor/team-management-rbac-configuring.md:6]: Ok, if you think the assumption is clear that way, I don't mind reducing the word count.

**Review by paigecalvert — APPROVED** [review]

**Pull request merged**